### DISCUSSION TITLE
What is the database.build project used for?

### DISCUSSION THREAD

**Discussion by carlsixsmith-moj** [kgrzybek/modular-monolith-with-ddd#300] [Q&A]
What is the reasoning behind the CompanyName.MyMeetings.Database.Build project? It doesn't appear to be referenced anywhere?

Is this a left behind attempt at using the dotnet core version of db projects or something thats there for future work to replace the sqlproj?

**Comment by plopezllorca** [comment]
I had the same doubt, but it is explained in this section:

https://github.com/kgrzybek/modular-monolith-with-ddd?tab=readme-ov-file#sql-server-database-project-build
